src/index.html

```diff
@@ -9,9 +9,160 @@
+            name="age"
+          >
+        </label>
+
+        <label class="form-element">Full date of birth:
+          <input
+            type="date"
+            name="birthDate"
+          >
+        </label>
+
+        <label class="form-element">I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="policyAgreement"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="form-group">
+        <legend>Registration</legend>
+
+        <label class="form-element">E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+          >
+        </label>
+
+        <label class="form-element">Password:
+          <input
+            type="text"
+            name="password"
+            minlength="7"
+            maxlength="25"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="form-group">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="form-element">Do you love cats?
+
+          <label>
+            <input
+              type="radio"
+              value="yes"
+              name="loveToCats"
+            >
+            Yes
+          </label>
+
+          <label>
+            <input
+              type="radio"
+              value="no"
+              name="loveToCats"
+            >
+            No
+          </label>
+        </div>
+
+        <label class="form-element">What is your favorite color?
+          <input
+            type="color"
+            name="colour"
+          >
+        </label>
+
+        <label class="form-element">What time do you go to bed?
+          <input
+            type="time"
+            name="timeForSleep"
+          >
+        </label>
+
+        <label class="form-element">What are your favorite brand of cars?
+          <select
+            name="favBrand"
+            multiple
+          >
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">Lada</option>
+          </select>
+        </label>
+
+        <label class="form-element">How do you rate our work?
+          <input
+            type="range"
+            name="workRate"
+            min="0"
+            max="10"
+            value="0"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="form-group">
+        <legend>Additional info:</legend>
+
+        <label class="form-element">Comments:
+          <textarea
+            name="comment"
+            cols="30"
+            rows="2"
+          >
+          </textarea>
+        </label>
+
+        <label class="form-element">Would you recommend us?
+          <select name="feedback">
+            <option value="yes" selected>yes</option>
+            <option value="no">no</option>
+          </select>
+        </label>
+      </fieldset>
+
+      
+      <button type="submit">Submit</button>
+      
+    </form>
```

Comment:
```suggestion
      </fieldset>

      <button type="submit">Submit</button>
    </form>
```
One empty line is enough. Also, don't leave empty lines between parent and child elements

Reply:
pls check now 
[DEMO LINK](https://YarikDubar.github.io/layout_html-form/)
- [TEST REPORT LINK](https://YarikDubar.github.io/layout_html-form/report/html_report/)

Reply:
its ok?